app/views/idv/sessions/step1.html.slim

```diff
@@ -0,0 +1,48 @@
+- title t('idv.titles.welcome')
```

Comment:
I think I meant `show` yes. So there would be no `render` call within the `show` method because the file name matches.